src/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+add_subdirectory(Interface)
+add_subdirectory(Optimization)
+add_subdirectory(LinAlg)
+add_subdirectory(Utils)
+add_subdirectory(Drivers)
+
```

Comment:
I would move this line to the very bottom of the CMake file. Everything that is in `Drivers` directory links to the HiOp library, so I would make sure the library is built and the aliases are set correctly before building the examples in the `Drivers` dir. I think CMake has its way in managing order in which different targets are built, but I would just make that order explicit in this code.

Reply:
Again, good point